integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManagedCoherence.java

```diff
@@ -349,17 +327,18 @@ private boolean coherenceCacheTest(String hostName) {
   private boolean coherenceCacheTest(String hostName, int ingressServiceNodePort) {
     logger.info("Starting to test the cache");
 
-    String hostAndPort = (OKD) ? hostName : K8S_NODEPORT_HOST + ":" + ingressServiceNodePort;
+    String hostAndPort = getHostAndPort(hostName, ingressServiceNodePort);
     logger.info("hostAndPort = {0} ", hostAndPort);
 
+
     // add the data to cache
     String[] firstNameList = {"Frodo", "Samwise", "Bilbo", "peregrin", "Meriadoc", "Gandalf"};
     String[] secondNameList = {"Baggins", "Gamgee", "Baggins", "Took", "Brandybuck", "TheGrey"};
     ExecResult result = null;
     for (int i = 0; i < firstNameList.length; i++) {
       result = addDataToCache(firstNameList[i], secondNameList[i], hostName, hostAndPort);
-      logger.info("Data added to the cache " + result.stdout());
       assertTrue(result.stdout().contains(firstNameList[i]), "Did not add the expected record");
+      logger.info("Data added to the cache " + result.stdout());
```

Comment:
change to: logger.info("Data added to the cache: {0}", result.stdout());

Reply:
fixed